universal-login-example/src/views/BackupView.js

```diff
@@ -8,30 +8,60 @@ class BackupView extends Component {
       <div className="subview">
         <div className="container">
           <h1 className="main-title">BACKUP CODES</h1>
-          <div className="row align-items-center">
-            <Blockies seed="alwaysUse.toLowerCase" size={8} scale={6} />
-            <div>
-              <p className="user-id">bobby.universal-id.eth</p>
-              <p className="wallet-address">
-                0xcee7a4d8be1c30623adc6185b6cdbcba19fac166
-              </p>
-            </div>
-          </div>
           <p className="backup-text">
-            Keep this codes somewhere safe and secret. Also, don&apos;t forget
-            your username as it&apos;s required to recover access. These are
-            independent codes, use each one once. Keep offline and away from
-            computers.
+            Print these, cut them apart and keep them safe locations apart from
+            each other. Keep them away from computers until you want to use
+            them.
           </p>
           <hr className="separator-s" />
-          <p className="backup-code bold">bamdaa-ewar-izoisi</p>
+          <div className="row align-items-center">
```

Comment:
we should not hard code 3 items, instead we should have an unbounded array and add 3 more at each time the user clicks to generate more codes

Reply:
Will make this change